balena/twofactor_auth.py

```diff
@@ -113,16 +115,16 @@ def generate_code(self, secret):
         totp = pyotp.TOTP(secret)
         return totp.now()
 
-    def get_otpauth_secret(self):
```

Comment:
Since we are changing the method name should this be  major changes?
Moreover, I still see this being used in the examples, even though we renamed it.

Reply:
@thgreasi, Updated. I will merge this to `v11` branch and release in next major release.